src/javascript/app/pages/cashier/cashier.js

```diff
@@ -252,7 +252,22 @@ const Cashier = (() => {
                     $('#view_payment_methods').attr('href', previous_href.concat('?anchor=cryptocurrency'));
                 } else {
                     $('.normal_currency').setVisibility(1);
+                    $('#change-account-btn').off('click').on('click', (e)=>{
+                        $('.main-account').trigger('click');
+                        $('html, body').animate({ scrollTop: 0 }, 'slow');
+                        e.stopPropagation();
```

Comment:
Since you're trying to prevent the event from bubbling, shouldn't you place the `e.stopPropagation();` before executing the other trigger / functions?
```suggestion
                    $('#change-account-btn').off('click').on('click', (e)=>{
                        e.stopPropagation();
                        $('.main-account').trigger('click');
                        $('html, body').animate({ scrollTop: 0 }, 'slow');
```

Reply:
changed according to review